metricshub-web/react/src/components/status/otel-status-icon.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Box, CircularProgress, Tooltip } from "@mui/material";
+import { useAppSelector } from "../../hooks/store";
+import otelColor from "../../assets/opentelemetry-icon-color.png";
+
+/**
+ * OpenTelemetry status icon component
+ * @param {object} props - Component props
+ * @param {object} props.sx - MUI sx prop for styling
+ * @returns JSX.Element
+ */
+export default function OtelStatusIcon({ sx = {} }) {
```

Comment:
This component should show up only when the OpenTelemetry Collector is installed.

In MetricsHub Community, the OpenTelemetry Collector is not installed and not spawned by MetricsHub, so there is no need to display the Otel icon.